### ISSUE TITLE
ENOENT: no such file or directory, open

### ISSUE THREAD

**Issue by evdama** [wdtamagi/svelte-avatar#2]
I'm trying to use it with a Sapper project with the standard rollup config. I  installed via `npm install --save-dev svelte-avatar` but getting this error:

```
Error: Could not load /Users/sa/0/edm/node_modules/svelte-avatar/src/lib/Avatar.svelte (imported by src/routes/chat.svelte): ENOENT: no such file or directory, open '/Users/sa/0/edm/node_modules/svelte-avatar/src/lib/Avatar.svelte'
```

The installed svelte-avatar directory in node_modules is
![Screen Shot 2020-07-19 at 14 38 00](https://user-images.githubusercontent.com/39374084/87874875-a3189f80-c9cd-11ea-8c0c-f7e5ad2cc68a.png)
So it's inside a `dist` folder and not inside `src/lib` as the error suggests.

**Comment by wdtamagi** [comment]
Sorry about the delay, i will work on that.

Thanks for report!

**Comment by wdtamagi** [comment]
Fixed by d5e3628aa96f7820e56ef473a7595fe67333a0e4
And generated version 1.0.6 on npm
